stacks/api/main.ts

```diff
@@ -102,6 +102,13 @@ const args = yargs
       default: 10
     }
   })
+  .options({
+    "function-timeout": {
+      number: true,
+      description: "The timeout value to be reached before kill the process",
+      default: 60000
+    }

```

Comment:
There a few things to correct here.

1- Can we just put this option into the same group with function-pool-size cause these options belong to function module.
2- Change description to: `Amount of time in milliseconds that has to elapse before aborting a function.`